core/src/main/java/io/knotx/databridge/core/DataBridgeKnotProxy.java

```diff
@@ -15,80 +15,52 @@
  */
 package io.knotx.databridge.core;
 
-import io.knotx.databridge.core.impl.DataBridgeSnippet;
 import io.knotx.databridge.core.impl.FragmentProcessor;
-import io.knotx.knotengine.api.AbstractKnotProxy;
-import io.knotx.knotengine.api.SnippetFragment;
-import io.knotx.knotengine.api.SnippetFragmentsContext;
-import io.knotx.server.api.context.ClientResponse;
-import io.netty.handler.codec.http.HttpResponseStatus;
-import io.reactivex.Observable;
+import io.knotx.engine.api.FragmentEvent;
+import io.knotx.engine.api.FragmentEventContext;
+import io.knotx.engine.api.FragmentEventResult;
+import io.knotx.engine.api.TraceableKnotOptions;
+import io.knotx.engine.api.TraceableKnotProxy;
+import io.reactivex.Maybe;
 import io.reactivex.Single;
 import io.vertx.core.logging.Logger;
 import io.vertx.core.logging.LoggerFactory;
 import io.vertx.reactivex.core.Vertx;
-import java.util.Collections;
-import java.util.Optional;
-import java.util.Set;
 
-public class DataBridgeKnotProxy extends AbstractKnotProxy {
+public class DataBridgeKnotProxy extends TraceableKnotProxy {
 
   private static final Logger LOGGER = LoggerFactory.getLogger(DataBridgeKnotProxy.class);
-
-  public static final String SUPPORTED_FRAGMENT_ID = "databridge";
+  public static final String DEFAULT_SUCCESS_TRANSITION = "next";
```

Comment:
Maybe there should be constant `TraceableKnotProxy.DEFAULT_SUCCESS_TRANSITION` containing `"next"`?